.dntrc

```diff
@@ -0,0 +1,19 @@
+## DNT config file
+## see https://github.com/rvagg/dnt
+
+NODE_VERSIONS="\
+  master   \
+  v7.x-staging  \
+  v6.x-staging  \
+  v4.x-staging  \
```

Comment:
@rvagg I picked these branches because I thought it might be interesting to test on running targets. Feel free to give some input here :)